programmers2/음양-더하기(reduce).js

```diff
@@ -0,0 +1,5 @@
+function solution(absolutes, signs) {
+    return absolutes.reduce((acc, value, curIndex) => {
+                        if (!signs[curIndex]) value *= -1 ;
+                        return acc + value ;}, 0);
```

Comment:
1.줄바꿈도 컨벤션입니다. 요런느낌으로 해주시면 돼여

2.value *= -1 ; 이부분을
return acc - value; 요런식으로도 가능합니다

3.
```js 
return absolutes.reduce((acc, value, curIndex) => {
        return signs[curIndex] ? acc + value : acc - value
    }, 0);
```
이런식으로 삼항을 사용하는 방법도 가능합니다.
if문을 선호하는 회사들도 많지만 프론트 개발을 하다보면
어쩔 수 없이 삼항을 사용해야 되는 상황이 생기기도 하여 알아두면 좋을 것 같아요!
```suggestion
    return absolutes.reduce((acc, value, curIndex) => {
      if (!signs[curIndex]) return acc - value;
      return acc + value ;
      }, 0);
```